recipes/edgetest-conda/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "edgetest-conda" %}
+{% set version = "2021.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 35dbfd259f9de56f9e4d9287fad09c22633a5e70992fa6533455e3fb124182cf
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install -vv --no-deps ."
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - edgetest
+    - conda
+    - setuptools
```

Comment:
It doesn't look like this is actually needed at runtime? You could move it to `host` if you like since it is used by `setup.py`.